Code/GraphMol/MolDraw2D/Wrap/testMolDraw2D.py

```diff
@@ -185,8 +185,12 @@ def testDrawMoleculesArgs(self):
                          highlightAtomColors=acolors)
     drawer.FinishDrawing()
     svg = drawer.GetDrawingText()
+    with open('testDrawMoleculeArgs_1.svg', 'w') as f:
+      f.write(svg)
     # 4 molecules, 6 bonds each:
-    self.assertEqual(svg.count('fill:none;fill-rule:evenodd;stroke:#FF7F7F'), 24)
+    re_str = r"path class='bond-\d+ atom-\d+ atom-\d+' d='M \d+.\d+,\d+.\d+ L \d+.\d+,\d+.\d+ L \d+.\d+,\d+.\d+ L \d+.\d+,\d+.\d+ Z' style='fill:#FF7F7F;"
```

Comment:
oh gracious... another regex.
:nauseated_face:

Reply:
I am getting quite attached to them for test purposes.